app/views/assignments/index.html.erb

```diff
@@ -0,0 +1,2 @@
+
+<%= react_component("ActionItemDashboard", {}) %>
```

Comment:
I'll ask about this during worksession tomorrow. A little confused on who's supposed to see this page